<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>717</width>
    <height>395</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tab_main">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>0</y>
      <width>591</width>
      <height>381</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab_5">
     <attribute name="title">
      <string>Home</string>
     </attribute>
     <widget class="QGroupBox" name="gb_home_1">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>30</y>
        <width>221</width>
        <height>111</height>
       </rect>
      </property>
      <property name="title">
       <string/>
      </property>
      <widget class="QLabel" name="label_2">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>20</y>
         <width>88</width>
         <height>81</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap resource="photo.qrc">:/icons/download2.png</pixmap>
       </property>
      </widget>
      <widget class="QLabel" name="label_8">
       <property name="geometry">
        <rect>
         <x>80</x>
         <y>20</y>
         <width>121</width>
         <height>71</height>
        </rect>
       </property>
       <property name="text">
        <string>Download Any Type Of Files</string>
       </property>
       <property name="wordWrap">
        <bool>true</bool>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="gb_home_2">
      <property name="geometry">
       <rect>
        <x>320</x>
        <y>30</y>
        <width>221</width>
        <height>111</height>
       </rect>
      </property>
      <property name="title">
       <string/>
      </property>
      <widget class="QLabel" name="label_5">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>20</y>
         <width>88</width>
         <height>81</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap resource="photo.qrc">:/icons/video2.png</pixmap>
       </property>
      </widget>
      <widget class="QLabel" name="label_9">
       <property name="geometry">
        <rect>
         <x>90</x>
         <y>20</y>
         <width>121</width>
         <height>71</height>
        </rect>
       </property>
       <property name="text">
        <string>Aparat Video Download</string>
       </property>
       <property name="wordWrap">
        <bool>true</bool>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="gb_home_4">
      <property name="geometry">
       <rect>
        <x>320</x>
        <y>180</y>
        <width>221</width>
        <height>111</height>
       </rect>
      </property>
      <property name="title">
       <string/>
      </property>
      <widget class="QLabel" name="label_6">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>10</y>
         <width>96</width>
         <height>96</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap resource="photo.qrc">:/icons/setting2.png</pixmap>
       </property>
      </widget>
      <widget class="QLabel" name="label_11">
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>20</y>
         <width>121</width>
         <height>71</height>
        </rect>
       </property>
       <property name="text">
        <string>App Themes &amp; Settings</string>
       </property>
       <property name="wordWrap">
        <bool>true</bool>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="gb_home_3">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>180</y>
        <width>221</width>
        <height>111</height>
       </rect>
      </property>
      <property name="title">
       <string/>
      </property>
      <widget class="QLabel" name="label_7">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>10</y>
         <width>77</width>
         <height>85</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap resource="photo.qrc">:/icons/video_playlist.png</pixmap>
       </property>
       <property name="scaledContents">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QLabel" name="label_10">
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>20</y>
         <width>121</width>
         <height>71</height>
        </rect>
       </property>
       <property name="text">
        <string>Aparat Playlist Download</string>
       </property>
       <property name="wordWrap">
        <bool>true</bool>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>File Download</string>
     </attribute>
     <widget class="QPushButton" name="btn_main_dl">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>270</y>
        <width>131</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>TeX Gyre Termes Math</family>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Start Download</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="le_main_url">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>40</y>
        <width>551</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>TeX Gyre Termes Math</family>
       </font>
      </property>
      <property name="placeholderText">
       <string>Enter Download link here</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="le_main_savelocation">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>110</y>
        <width>491</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>TeX Gyre Termes Math</family>
       </font>
      </property>
      <property name="placeholderText">
       <string>Enter Save location ( use B button)</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_main_browse">
      <property name="geometry">
       <rect>
        <x>530</x>
        <y>110</y>
        <width>51</width>
        <height>41</height>
       </rect>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset resource="photo.qrc">
        <normaloff>:/icons/dir.png</normaloff>:/icons/dir.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>32</width>
        <height>32</height>
       </size>
      </property>
      <property name="flat">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QProgressBar" name="pb_main">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>190</y>
        <width>551</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>TeX Gyre Termes Math</family>
       </font>
      </property>
      <property name="value">
       <number>0</number>
      </property>
      <property name="format">
       <string>%p%</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Video Download</string>
     </attribute>
     <widget class="QTabWidget" name="tabWidget_2">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>691</width>
        <height>351</height>
       </rect>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab_3">
       <attribute name="title">
        <string>One Video</string>
       </attribute>
       <widget class="QPushButton" name="pushButton_2">
        <property name="geometry">
         <rect>
          <x>500</x>
          <y>20</y>
          <width>51</width>
          <height>41</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>TeX Gyre Termes Math</family>
         </font>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="photo.qrc">
          <normaloff>:/icons/check.png</normaloff>:/icons/check.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>32</width>
          <height>32</height>
         </size>
        </property>
        <property name="flat">
         <bool>true</bool>
        </property>
       </widget>
       <widget class="QPushButton" name="btn_onevideo_browse">
        <property name="geometry">
         <rect>
          <x>500</x>
          <y>80</y>
          <width>51</width>
          <height>41</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>TeX Gyre Termes Math</family>
         </font>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="photo.qrc">
          <normaloff>:/icons/dir.png</normaloff>:/icons/dir.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>32</width>
          <height>32</height>
         </size>
        </property>
        <property name="flat">
         <bool>true</bool>
        </property>
       </widget>
       <widget class="QLineEdit" name="le_onevideo_saveloaction">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>80</y>
          <width>471</width>
          <height>41</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>TeX Gyre Termes Math</family>
         </font>
        </property>
        <property name="placeholderText">
         <string>Enter Save location (use B button)</string>
        </property>
       </widget>
       <widget class="QPushButton" name="btn_onevideo_dl">
        <property name="geometry">
         <rect>
          <x>220</x>
          <y>260</y>
          <width>130</width>
          <height>41</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>TeX Gyre Termes Math</family>
          <pointsize>11</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Start Download</string>
        </property>
       </widget>
       <widget class="QLabel" name="lbl_onevideo_vidquality">
        <property name="geometry">
         <rect>
          <x>110</x>
          <y>150</y>
          <width>91</width>
          <height>31</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>TeX Gyre Termes Math</family>
         </font>
        </property>
        <property name="text">
         <string>Video Quality</string>
        </property>
       </widget>
       <widget class="QProgressBar" name="pb_onevideo">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>210</y>
          <width>540</width>
          <height>31</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>TeX Gyre Termes Math</family>
         </font>
        </property>
        <property name="value">
         <number>0</number>
        </property>
       </widget>
       <widget class="QComboBox" name="cbb_onevideo_vidquality">
        <property name="geometry">
         <rect>
          <x>250</x>
          <y>150</y>
          <width>241</width>
          <height>26</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>TeX Gyre Termes Math</family>
         </font>
        </property>
       </widget>
       <widget class="QLineEdit" name="le_onevideo_url">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>20</y>
          <width>471</width>
          <height>41</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>TeX Gyre Termes Math</family>
         </font>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="placeholderText">
         <string>Enter video URL</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_4">
       <attribute name="title">
        <string>Full Playlist</string>
       </attribute>
       <widget class="QLineEdit" name="le_fullplaylist_url">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>10</y>
          <width>540</width>
          <height>40</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>TeX Gyre Termes Math</family>
         </font>
        </property>
        <property name="placeholderText">
         <string>Enter Playlist URL</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="le_fullplaylist_savelocation">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>70</y>
          <width>470</width>
          <height>41</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>TeX Gyre Termes Math</family>
         </font>
        </property>
        <property name="placeholderText">
         <string>Enter Save location ( use B button)</string>
        </property>
       </widget>
       <widget class="QPushButton" name="btn_fullplaylist_browse">
        <property name="geometry">
         <rect>
          <x>500</x>
          <y>70</y>
          <width>51</width>
          <height>41</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>TeX Gyre Termes Math</family>
         </font>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="photo.qrc">
          <normaloff>:/icons/dir.png</normaloff>:/icons/dir.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>32</width>
          <height>32</height>
         </size>
        </property>
        <property name="flat">
         <bool>true</bool>
        </property>
       </widget>
       <widget class="QLabel" name="lbl_fullplaylist_vidquality">
        <property name="geometry">
         <rect>
          <x>110</x>
          <y>140</y>
          <width>91</width>
          <height>31</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>TeX Gyre Termes Math</family>
         </font>
        </property>
        <property name="text">
         <string>Video Quality</string>
        </property>
       </widget>
       <widget class="QComboBox" name="cbb_fullplaylist_vidquality">
        <property name="geometry">
         <rect>
          <x>250</x>
          <y>140</y>
          <width>241</width>
          <height>26</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>TeX Gyre Termes Math</family>
         </font>
        </property>
       </widget>
       <widget class="QProgressBar" name="pb_fullplaylist">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>230</y>
          <width>540</width>
          <height>21</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>TeX Gyre Termes Math</family>
         </font>
        </property>
        <property name="value">
         <number>0</number>
        </property>
       </widget>
       <widget class="QPushButton" name="btn_fullplaylist_dl">
        <property name="geometry">
         <rect>
          <x>220</x>
          <y>270</y>
          <width>130</width>
          <height>41</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>TeX Gyre Termes Math</family>
          <pointsize>11</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Start Download</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_3">
        <property name="geometry">
         <rect>
          <x>110</x>
          <y>195</y>
          <width>90</width>
          <height>21</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>TeX Gyre Termes Math</family>
         </font>
        </property>
        <property name="text">
         <string>Current Video:</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_4">
        <property name="geometry">
         <rect>
          <x>320</x>
          <y>195</y>
          <width>90</width>
          <height>21</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>TeX Gyre Termes Math</family>
         </font>
        </property>
        <property name="text">
         <string>Playlist Video:</string>
        </property>
       </widget>
       <widget class="QLCDNumber" name="lcdNumber">
        <property name="geometry">
         <rect>
          <x>180</x>
          <y>190</y>
          <width>61</width>
          <height>23</height>
         </rect>
        </property>
        <property name="frameShape">
         <enum>QFrame::NoFrame</enum>
        </property>
       </widget>
       <widget class="QLCDNumber" name="lcdNumber_2">
        <property name="geometry">
         <rect>
          <x>370</x>
          <y>190</y>
          <width>64</width>
          <height>23</height>
         </rect>
        </property>
        <property name="frameShape">
         <enum>QFrame::NoFrame</enum>
        </property>
       </widget>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_6">
     <attribute name="title">
      <string>Settings</string>
     </attribute>
     <widget class="QPushButton" name="pushButton">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>170</y>
        <width>141</width>
        <height>91</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>50</y>
        <width>231</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Choose Your Theme</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_3">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>170</y>
        <width>141</width>
        <height>91</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_4">
      <property name="geometry">
       <rect>
        <x>380</x>
        <y>170</y>
        <width>141</width>
        <height>91</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QPushButton" name="btn_home">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>101</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="photo.qrc">
      <normaloff>:/icons/home.png</normaloff>:/icons/home.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>63</width>
      <height>63</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_filedl">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>90</y>
      <width>101</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="photo.qrc">
      <normaloff>:/icons/dl_dir.png</normaloff>:/icons/dl_dir.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>63</width>
      <height>63</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_videodl">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>170</y>
      <width>101</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="photo.qrc">
      <normaloff>:/icons/video.png</normaloff>:/icons/video.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>63</width>
      <height>63</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_settings">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>250</y>
      <width>101</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="photo.qrc">
      <normaloff>:/icons/setting.png</normaloff>:/icons/setting.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>63</width>
      <height>63</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_quit">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>340</y>
      <width>101</width>
      <height>32</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>TeX Gyre Termes Math</family>
      <pointsize>11</pointsize>
      <weight>75</weight>
      <bold>true</bold>
      <stylestrategy>PreferAntialias</stylestrategy>
     </font>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 0, 0);</string>
    </property>
    <property name="text">
     <string>Quit</string>
    </property>
    <property name="icon">
     <iconset resource="photo.qrc">
      <normaloff>:/icons/exit.png</normaloff>:/icons/exit.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>40</width>
      <height>27</height>
     </size>
    </property>
    <property name="checkable">
     <bool>false</bool>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>717</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="photo.qrc"/>
 </resources>
 <connections/>
</ui>
